### PR TITLE
Adds a basic ci config to run lint and build

### LINKED ISSUE #146: Add a basic CI setup
Add a CI setup that checks
- [ ] Angular Lint Task
- [ ] Angular Build Task

### PR DESCRIPTION
**Pull request by kelsos**
@palango do you have admin access to the repository?

CircleCI is not enabled for the explorer repository and I don't have access to enable it.

Closes #146

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          paths:
+            - "node_modules"
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./*
+
+  lint:
+    executor: base-executor
+    working_directory: ~/src/frontent
```
Comment (palango) [inline .circleci/config.yml]: This should be fronten*d*, wonder hwo this works ;)

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    steps:
+      - attach_workspace: *attach_options
+      - run: npm run lint
+
+  build:
+    executor: base-executor
+    working_directory: ~/src/frontent
```
Comment (palango) [inline .circleci/config.yml]: same here

File: .circleci/config.yml (hunk 1 above)
Comment (kelsos) [reply .circleci/config.yml]: OK let me see if it works after removing them completely then :+1:

Comment (kelsos) [reply .circleci/config.yml]: It is probably because this is also setup in the executor

Comment (kelsos) [reply .circleci/config.yml]: but it seems that the install part still needs the working_directory specified or it will fail.

**Pull request merged**